### DISCUSSION TITLE
Where to add collection permit information?

### DISCUSSION THREAD

**Discussion by AFisherCSULB** [Symbiota/community-board#89] [Q&A]
I'd like to begin listing collection permit numbers to the occurrence record of new specimens in the Long Beach Herbarium. What would be the best field for this information? Would MISC<Notes be the appropriate place? I'd like to follow a standardized approach so I can search later by permit.

**Comment by arbolitoloco** [comment]
Hi @AFisherCSULB, as far as I know (but other might correct me) there is no standard approach for permit numbers yet, but I have a suggestion:

1. Add it to the occurrence record as an additional catalog number (field `Other Cat. #s` in the occurrence editor form, or `otherCatalogNumbers` in CSV), making it clear that it's a permit number. Like so:
```
Collection Permit Number: 123456-2019
```
Please keep in mind that adding the permit number to the `otherCatalogNumbers` field will make it publicly available in the downloads and searches (I don't know if you have any issues with that number being public).

2. Search the specimens that share the same permit number and add them to a `Dataset`, by clicking the dataset management icon, as shown in the gif below:

![seinet-occ-dataset-demo](https://user-images.githubusercontent.com/17555247/151857704-812a3ec8-b321-4446-9efe-f0d46a27712c.gif)
 
That way, you have a place to "organize" your records in a collection, and in the future you can change how your permit numbers are recorded, if needed.
